bindings/js/gen.go

```diff
@@ -0,0 +1,90 @@
+package js
+
+import (
+	"bytes"
+	"embed"
+	_ "embed"
+	"fmt"
+	"net/url"
+	"path"
+	"text/template"
+
+	"github.com/onflow/flixkit-go/common"
+)
+
+type SimpleParameter struct {
+	Name string
+	Type string
+}
+
+type TemplateData struct {
+    Version     string
+    Parameters []SimpleParameter
+    Title       string
+    Location    string
+    IsScript    bool
+    IsLocalTemplate bool
```

Comment:
See the above comment, but yeah you shouldn't need to know local here. Grab it in the CLI and pass it in like the examples show for running FLIX in the CLI

Reply:
Reason for local check is reference in javascript is different depending on if the template is local. 

import local file
import flixTemplate from "{{.Location}}"

or if it's a url. 
const flixTemplate = "{{.Location}}"